tests/Unit/DataStructures/Tensor/Expressions/Test_MixedOperations.cpp

```diff
@@ -133,11 +157,149 @@ void test_mixed_operations(const DataType& used_for_size) noexcept {
+  // \f$N = \sqrt{g^{ij} * \psi_{jt} * \psi_{it} - \psi_{tt}}\f$
+  const Scalar<DataType> actual_result_tensor_returned =
+      TensorExpressions::evaluate(sqrt(
+          g(ti_I, ti_J) * psi(ti_j, ti_t) * psi(ti_i, ti_t) - psi(ti_t, ti_t)));
+  Scalar<DataType> actual_result_tensor_filled{};
+  TensorExpressions::evaluate(
+      make_not_null(&actual_result_tensor_filled),
+      sqrt(g(ti_I, ti_J) * psi(ti_j, ti_t) * psi(ti_i, ti_t) -
+           psi(ti_t, ti_t)));
+
+  CHECK_ITERABLE_APPROX(actual_result_tensor_returned.get(),
+                        expected_result_tensor.get());
+
+  // Test with TempTensor for LHS tensor
+  if constexpr (not std::is_same_v<DataType, double>) {
+    Variables<tmpl::list<::Tags::TempTensor<1, Tensor<DataType>>>>
+        actual_result_tensor_temp_var{used_for_size.size()};
+    Scalar<DataType>& actual_result_tensor_temp =
+        get<::Tags::TempTensor<1, Tensor<DataType>>>(
+            actual_result_tensor_temp_var);
+    ::TensorExpressions::evaluate(
+        make_not_null(&actual_result_tensor_temp),
+        sqrt(g(ti_I, ti_J) * psi(ti_j, ti_t) * psi(ti_i, ti_t) -
+             psi(ti_t, ti_t)));
+
+    CHECK_ITERABLE_APPROX(actual_result_tensor_temp.get(),
+                          expected_result_tensor.get());
+  }
+}
+
+// Includes an expression with addition, subtraction, an inner product, outer
+// products, a scalar, and two calls to `evaluate` that fill the time and
+// spatial components of the result tensor, respectively
+//
+// Note: This is the calculation of the spacetime derivative of the spacetime
+// metric, \f$\partial_c g_{ab}\f$
+template <typename DataType, typename Generator>
+void test_case3(const DataType& used_for_size,
+                const gsl::not_null<Generator*> generator) noexcept {
+  std::uniform_real_distribution<> distribution(0.1, 1.0);
+
+  const auto alpha = make_with_random_values<Scalar<DataType>>(
+      generator, make_not_null(&distribution), used_for_size);
+
+  const auto beta =
+      make_with_random_values<tnsr::I<DataType, 3, Frame::Inertial>>(
+          generator, make_not_null(&distribution), used_for_size);
+
+  const auto pi =
+      make_with_random_values<tnsr::aa<DataType, 3, Frame::Inertial>>(
+          generator, make_not_null(&distribution), used_for_size);
+
+  const auto phi =
+      make_with_random_values<tnsr::iaa<DataType, 3, Frame::Inertial>>(
+          generator, make_not_null(&distribution), used_for_size);
+
+  using result_tensor_type = tnsr::abb<DataType, 3, Frame::Inertial>;
+  result_tensor_type expected_result_tensor =
+      compute_expected_result3(alpha, beta, pi, phi, used_for_size);
+  result_tensor_type actual_result_tensor_filled{};
+  // \f$\partial_t g_{ab} = -\alpha * \Pi_{ab} + \beta^i * \Phi_{iab}\f$
+  TensorExpressions::evaluate<ti_t, ti_a, ti_b>(
+      make_not_null(&actual_result_tensor_filled),
+      -1.0 * alpha() * pi(ti_a, ti_b) + beta(ti_I) * phi(ti_i, ti_a, ti_b));
```

Comment:
why do we need the `-1.0` instead of `-alpha()`?

Reply:
The unary `-` operation is not currently defined for TEs. Is this something you would like me to add in a follow up PR?